src/components/molecules/Settings/ProjectList/ProjectCell/index.tsx

```diff
@@ -73,6 +75,7 @@ const Wrapper = styled.div<{ isHover?: boolean }>`
   cursor: pointer;
   height: 100%;
   background-color: ${props => (props.isHover ? props.theme.main.lightTransparentBg : "")};
+  border: ${props => (props.isHover ? " 2px solid " + props.theme.main.select : " ")};
```

Comment:
Because you are adding a border on hover, when you hover the content gets pushed in by 2px on each side. In this case, its the project name, description and publication status.
One option is to always have `border: 2px solid`, and then set `border-color` is set on hover. To keep dimensions accurate this way, you might have to change margins/padding for the content too. 
If you know a better method go for it!